docs/docs/how-to/dbauth-passwordless.md

```diff
@@ -0,0 +1,644 @@
+# Setting up dbAuth to be passwordless
+
+Security is really important.  Sometimes you don't want to integrate with a third-party authentication services.  Whatever the reason, Redwood has you covered with Redwood's dbAuth to authenticate users.  This is a great option.
+
+One thing though is now you're collecting the user's login and password.  If you'd like to not collect that, an alternative is to generate a token in place of the password.  The only data needed for passwordless is the users email address.
+
+In this how-to I'll show you how to set up dbAuth to be passwordless, you'll still need to set up a a way to send [send emails](../how-to/sending-emails.md), but there's plenty of ways to do that.
+
+## Background
+
+Let me start by sharing a little bit about how passwordless works.
+### What is a passwordless authentication method?
+A passwordless authentication method is a method of authentication where the user is not required to enter a password.  Instead, the user is sent a link to their email address.  When they click the link, they are logged in.
+
+Passwordless uses a token that is time-sensitive.  So instead of storing a password, we store a token, and an expiration.
+
+That token is generated randomly and is stored in the database.
+## How to do it
+### 1. Modify the Prisma schema
+First, we need to modify the Prisma schema.
+
+If you followed the tutorial you'll have a `User` model.  Here's is what it looks like with after the changes.
+
+```jsx {4-6}
+model User {
+  id                  Int       @id @default(autoincrement())
+  name                String?
+  email               String    @unique
+  loginToken          String
+  loginTokenExpiresAt DateTime?
+  salt                String?
+}
+```
+
+Make note of the optional `salt` field.
+
+Once you've made the changes, you'll need to migrate your database.
+
+```bash
+yarn rw prisma migrate dev
+```
+
+### 2. Setting up the generateToken function
+Next, we need to create a function that will generate a token and an expiration date.
+
+If you followed the tutorial, you might not have a `/api/src/services/users/users.js` file.  If that's the case, you can create it with the following command using your terminal.
+
+```bash
+yarn rw g service users
+```
+
+Now that you have the file, let's add the `generateToken` function.
+
+```javascript {21} title="/api/src/services/users/users.js"
+// add this to the bottom of the file
+export const generateLoginToken = async ({ email }) => {
```

Comment:
There is a missing ``CryptoJS`` import. Where does it come from? If it comes from the ``crypto-js`` library, then it would be better to implement it using the ``Crypto`` module from NodeJS directly (if possible): https://nodejs.org/api/crypto.html